ansible/tasks/postgres-extensions/13-plv8.yml

```diff
@@ -37,6 +37,13 @@
   when: platform == "arm64"
   ignore_errors: yes # not needed for docker build
 
+- name: plv8 - enable ccache
+  become: yes
+  replace:
+    path: /tmp/plv8/Makefiles/Makefile.docker
+    regexp: '^GN_ARGS ='
+    replace: GN_ARGS = cc_wrapper="env CCACHE_SLOPPINESS=time_macros ccache"
```

Comment:
Using instructions from macos which also applies to linux https://chromium.googlesource.com/chromium/src/+/HEAD/docs/ccache_mac.md